jcl/src/java.base/share/classes/com/ibm/tools/attach/target/AttachHandler.java

```diff
@@ -119,6 +121,12 @@
 	/* [PR Jazz 30075] Make syncFileLock an instance variable since it is accessed only by the attachHandler singleton. */
 	FileLock syncFileLock;
 
+	static {
+		shutdownTimeoutMs = AccessController.doPrivileged((PrivilegedAction<Integer>) () -> {
```

Comment:
Do you have a test case showing this change is needed? 
- AttachHander is a bootstrap class used from a separate thread created at startup. I don't think this should be required
- using Lambda's during JVM startup affects startup performance negatively

Reply:
> Do you have a test case showing this change is needed?

No, the change was purely based on the static code review. Agreed this use scenario doesn't require `doPrivileged()`. Removed the change, please have another look.

Regarding Lambda's usages, a quick search `() ->` shows 600+ hit within `JDK11` extension repo `src` directory while there are other slightly different expressions. Unless there is a critical and significant performance impact, I would like take advantage of Lambda's benefits.

Reply:
See https://github.com/eclipse/openj9/issues/5373
There are no lamba's used in JVM bootstrap. Since you've already removed it, there is no problem.